tomcat/src/main/java/com/linecorp/armeria/server/tomcat/TomcatVersion.java

```diff
@@ -23,7 +23,10 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-final class TomcatVersion {
+import com.google.common.annotations.VisibleForTesting;
+
+@VisibleForTesting
+public final class TomcatVersion {
```

Comment:
Could you move this class to `com.linecorp.armeria.internal.tomcat` so it's not visible from Javadoc?

Reply:
`com.linecorp.armeria.tomcat.internal`?

Reply:
Our convention uses `com.linecorp.armeria.internal.*`. See `armeria-thrift` for an example.